adage/adage/settings.py

```diff
@@ -82,12 +82,18 @@
                     "tokenizer": "standard",
                     "filter": [
                         "standard",
-                        "word_delimiter",
+                        "adage_word_delimiter",
                         "lowercase",
                         "stop",
                         "snowball"
                     ]
                 },
+            },
+            'filter': {
+                'adage_word_delimiter': {
+                    'type': 'word_delimiter',
+                    'split_on_case_change': False
```

Comment:
Nice! I think this makes sense. I was wondering though, any reason for using double quotes for strings above and single quotes here?